code/modules/food_and_drinks/food/snacks.dm

```diff
@@ -291,11 +297,15 @@ All foods are distributed among various categories. Use common sense.
 			else
 				S.reagents.add_reagent(r_id, amount)
 
-/obj/item/reagent_containers/food/snacks/microwave_act(obj/machinery/microwave/M)
+/obj/item/reagent_containers/food/snacks/microwave_act(obj/machinery/microwave/M, var/quality_increase)
```

Comment:
take out the var/, it's implied in proc header/whatever it's called

Reply:
i'm just gonna remove it as a parameter because it's defined in obj/machinery/microwave meaning it's not needed to be passed to microwave_act